aiobotocore/client.py

```diff
@@ -206,7 +209,14 @@ def _register_legacy_retries(self, client):
         )
         unique_id = 'retry-config-%s' % service_event_name
         client.meta.events.register(
-            'needs-retry.%s' % service_event_name, handler, unique_id=unique_id
+            f"needs-retry.{service_event_name}", handler, unique_id=unique_id
+        )
+
+    def _register_eventbridge_events(
+        self, client, endpoint_bridge, endpoint_url
+    ):
+        super()._register_eventbridge_events(
+            client, endpoint_bridge, endpoint_url
         )
```

Comment:
I don't think this needs to be added, the impl from the super should be fine given nothing is being overridden

Reply:
Got it, so calling `super()` from create_client is fine.